mage_ai/cli/main.py

```diff
@@ -1,3 +1,6 @@
+from mage_ai.cli.utils import parse_arguments, parse_runtime_variables
+from mage_ai.data_preparation.variable_manager import get_global_variables
```

Comment:
Let's put this import statement in its if-else branch so that we don't need to import it for commands that don't use variables.